functions/make_plots.R

```diff
@@ -0,0 +1,304 @@
+make_plots <- function(metab.list, stoftest, pt, zscore_cutoff, xaxis_cutoff, ThisProbScore,
```

Comment:
>-  function make_plots works, but structure is too complex. Split into prepare_data and violin_plot functions

Totally agree, there is a lot of data prep going on, not just making plots :).

Reply:
I stopped with comments on this function, since I think you will go over it again in the future. Which will probably solve a lot of comments.